cidc_schemas/prism.py

```diff
@@ -356,6 +361,11 @@ def _process_field_value(
     in template schema, that allows for multi-processing of a single cell value.
     """
 
+    # skip nullable
+    if "allow_empty" in field_def:
```

Comment:
Nit: I would also check for `allow_empty`s value:
```suggestion
    if field_def.get("allow_empty"):
```